PlayTools/Controls/PTFakeTouch/NSObject+Swizzle.m

```diff
@@ -224,8 +243,6 @@ + (void)load {
     // Set idiom to iPad
     // [objc_getClass("UIDevice") swizzleInstanceMethod:@selector(userInterfaceIdiom) withMethod:@selector(hook_userInterfaceIdiom)];
     // [objc_getClass("UITraitCollection") swizzleInstanceMethod:@selector(userInterfaceIdiom) withMethod:@selector(hook_userInterfaceIdiom)];
-
-    [objc_getClass("VSSubscriptionRegistrationCenter") swizzleInstanceMethod:@selector(setCurrentSubscription:) withMethod:@selector(hook_setCurrentSubscription:)];
```

Comment:
> Is there any specific reason why this is removed?

Xcode said VSSubscription is not supported in MacCatalyst, so I can't build PlayTools targeting for MacCatalyst.

Reply:
> Xcode said VSSubscription is not supported in MacCatalyst, so I can't build PlayTools targeting for MacCatalyst.

Actually playtools is built for iOS, then binarily converted to MacCatalyst. In this way you can build without removing this.

This swizzle is introduced by https://github.com/PlayCover/PlayTools/pull/84 and I think it should remain.

Reply:
@XuYicong I didn't know that. I revert VSS things in PR branch.

Reply:
In case anyone wondering how to convert the binary, this is the script I've been using:

```shell
#!/bin/sh
echo "Converting to maccatalyst"
vtool \
	-set-build-version maccatalyst 11.0 14.0 \
	-replace -output \
	"PlayTools" \
	"PlayTools"

echo "Codesigning PlayTools"
codesign -fs- "PlayTools"
```